SPECS/initscripts.spec

```diff
@@ -1,6 +1,6 @@
 %global package_speccommit 77f95d7401e9a3f58eaa4a42cc48dab78bf1b504
 %global usver 9.49.41
-%global xsver 2
+%global xsver 2.1
```

Comment:
We don't touch xsver. You can add .1 directly in the Release tag, before %{?dist}